tests/conftest.py

```diff
@@ -1904,6 +1905,26 @@ def _awscli_pod_cleanup():
     return awscli_pod_obj
 
 
+@pytest.fixture()
+def test_directory_setup(request, awscli_pod_session):
+    return test_directory_setup_fixture(request, awscli_pod_session)
+
+
+def test_directory_setup_fixture(request, awscli_pod_session):
+    origin_dir, result_dir = setup_pod_directories(
+        awscli_pod_session, ["origin", "result"]
+    )
+    SetupDirs = namedtuple("SetupDirs", "origin_dir, result_dir")
```

Comment:
I think SetupDirs should be changed to setup_dirs

Reply:
`SetupDirs` is a namedtuple. A call to namedtuple returns a class - and the PEP8 convention for class names is CamelCase. It is not a standard variable, and I think it's important to preserve the distinguishment.